services/scan/src/store.ts

```diff
@@ -447,35 +491,46 @@ export class Store {
    * Records that batches have been backed up.
    */
   setScannerAsBackedUp(): void {
-    this.setBackupTimestamp(BackupKey.Scanner);
+    if (this.hasElection()) {
+      this.client.run(
+        'update election set scanner_backed_up_at = current_timestamp'
+      );
+    } else {
+      throw new Error('Unconfigured scanner cannot be backed up.');
+    }
   }
 
   /**
    * Records that CVRs have been backed up.
    */
   setCvrsAsBackedUp(): void {
-    this.setBackupTimestamp(BackupKey.Cvrs);
+    if (this.hasElection()) {
+      this.client.run(
+        'update election set cvrs_backed_up_at = current_timestamp'
+      );
+    } else {
+      throw new Error('Unconfigured scanner cannot have exported cvrs.');
+    }
   }
 
   /**
-   * Sets the timestamp of a backup
+   * Gets the timestamp for the last scanner backup
    */
-  private setBackupTimestamp(key: BackupKey): void {
-    this.client.run(
-      'insert or replace into backups (key, value) values (?, current_timestamp)',
-      key
-    );
+  private getScannerBackupTimestamp(): Iso8601Timestamp | undefined {
+    const row = this.client.one('select scanner_backed_up_at from election') as
+      | { scanner_backed_up_at: Iso8601Timestamp }
+      | undefined;
+    return row?.scanner_backed_up_at;
   }
 
   /**
-   * Gets a config value by key.
+   * Gets the timestamp for the last cvr export
    */
-  private getBackupTimestamp(key: BackupKey): Iso8601Timestamp | undefined {
-    const row = this.client.one<[Iso8601Timestamp]>(
-      'select value from backups where key = ?',
-      key
-    ) as { value: Iso8601Timestamp } | undefined;
-    return row?.value;
+  private getCvrBackupTimestamp(): Iso8601Timestamp | undefined {
+    const row = this.client.one('select cvrs_backed_up_at from election') as
```

Comment:
One last time, then I'm going to just assume you know: camel case.

Reply:
Safe assumption.